src/components/App.jsx

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
+import { Provider } from 'react-redux';
```

Comment:
### Optional
- [ ] It would be better if we format our PR nicely, we can do this by leaving a space between `-` and the `text`